### ISSUE TITLE
Update README.rst.

### ISSUE THREAD

**Issue by thatguyisjames** [Evolution0/bandcamp-dl#147]
**Python version:** 
python2-2.7.1
python3-3.6.5.1

**Bandcamp-dl version:**
bandcamp-downloader (0.0.8.post12)

**Bancamp-dl options:**  
default?

**url:**  

**options:**  

**Describe the issue:**
-------------------------
Installing on vanilla Fedora 28 Workstation Edition. 
Default configuration has Python2 and Python3 installed

also by default "pip" refers to package "python2-pip" for older python.
When using instructions "pip install bandcamp-downloader"...
It first fails needing sudo to install, after adding sudo... 
It installs successfully, but under 2.7.1 system.

Running "bandcamp-dl [URL]" comes back with a terminal full of python 2.7.1 errors.

I always run into issues getting bandcamp-dl to work.

Finally found that I can use pip3 or pip3.6 to install into the proper python sub directories/library folders.

Fix? 

Change README.rst to reflect working install method. 
"sudo pip3.6 install bandcamp-downloader"
or even adding $ or # 
"$ sudo pip3.6 install bandcamp-downloader"
"# pip3.6 install bandcamp-downloader"

**Comment by thatguyisjames** [comment]
Note: Just checked my Debian install on my pi... 

Looks like package python3-pip and python-pip is avail, but no python2-pip. 
But "apt-cache show" tells me...
python-pip requires "python:any (<<2.8)"
python3-pip requires "python3:any (>= 3.4~)"

Installing "python3-pip" gives me the "pip3" command

I think this helps show the difference and why pip3(Fedora, Debian) or pip3.6(Fedora) is the proper installation program.

**Comment by Evolution0** [comment]
I don't support versions of Python below 3.4 as stated in the description so that is your first issue.

I'm not sure if the sudo requirement is just a quirk of you attempting to install under Python 2 at first (Running pip install with elevated permissions is a no-no, not sure if they changed something to reflect this in Python 3) or something to do with your system, most distros should default to `pip install --user` when calling `pip install`

It should always be `pip3` on a multiple installation system, otherwise pip works fine, if you can run `pip3.6` then you should be able to run `pip3` just as well.

Unless for some reason you have Python 2 and multiple versions of Python 3 installed.. (Which is why specific `pip3.x` commands exist. Choosing the right one is up to the user at that point, I've no magic bullet for that.

**Comment by thatguyisjames** [comment]
I totally agree this is a case by case basis.

Just for myself, not knowing python very well. The README didn't really lead me down the path for success. where as, on 4 different systems now "sudo pip3 install bandcamp-downloader" works without issue. One of my systems did installed 'successfully' without elevated privilages, but bandcamp-dl was not a usable command. Redid it with sudo, I could then use it in the command line. 

I read the wiki page, and it very brushes over... "Select "Add Python 3.x to PATH" at the bottom of the installer." which is fine for windows, but Linux users don't get that option, and could be what is breaking this pip and pip3 thing. 

I think that's were my frustration stems from.

then maybe if the wiki page just had a note for Linux installs, where I didn't have to turn to google to learn about this. Just trying to save future installers the time I had to hunt down the answer.

**Comment by Evolution0** [comment]
Likely I'll change the readme and install instructions to have more options (Standalone executable to install with wget/curl etc) as well as add the brew install option for mac users which many people might not know exists.

Though `sudo pip3 install bandcamp-downloader` will be listed as the recommended option for linux.

Hopefully once I get to a point where I feel I don't need to work on bandcamp-dl anymore I'll do some packages for popular distros.

**Comment by ajudator** [comment]
I would like to second that recommendation for Linux. I executed the README option, and ran into some problems - that I thought that were related to permission issues and etc. I didn't really pay attention to the Python version, when I realized it, I installed Python3 & pip3 and ran the recommendation with `pip3`. But only after reading this issue and sudoing pip3 I got out of the `bandcamp-dl: command not found` bash error and it was a success :smile: 

OK, I'm a noob with python and it was my fault about the Python version :joy:  I don't know if it is possible or wanted, but I really agree that the `sudo pip3 install bandcamp-downloader` fix sounds like a good solution (including the not-so-bright users like some :eyes: )
